python/lsst/daf/butler/registry/datasets/byDimensions/_manager.py

```diff
@@ -115,6 +115,22 @@ def refresh(self, *, universe: DimensionUniverse) -> None:
         self._byName = byName
         self._byId = byId
 
+    def deleteDatasetType(self, name: str) -> None:
```

Comment:
Not really sure what name is wanted here. prune vs delete vs pruneDatasetType vs deleteDatsetType. Should it be consistent with the registry method?

Reply:
I'd at least use the same name here as the registry method.  I think I used `prune` in `Butler.pruneDatasets` and `pruneCollections` because I wanted to signify partial deletion, e.g. untagging from a collection or unstoring from a Datastore, as I figured those would be the most common modes.  It looks like "remove" is beating "delete" 2-1 in other Registry precedent, and the "delete" (for `deleteOpaqueData`) is maybe a justifiable exception because it acts on "data" rather than countable entities.  So maybe "remove"?  I don't have a strong preference, but I gather you don't either.

Reply:
`removeDatasetType` is fine with me. I did wonder whether `remove` would be preferred in the manager because you have a `register` there and not a `registerDatasetType`.

Reply:
Ah, yeah, `remove` alone is probably a better match for the storage, given that `register` symmetry (I was only thinking about the initial verb).

Reply:
Agree `remove` should be fine.